api/api/specs/common.yaml

```diff
@@ -29,6 +29,7 @@ components:
       properties:
         name:
           type: "string"
+          pattern: '^[a-zA-Z0-9_]*$'
```

Comment:
The regex expressions look slightly different from those of the UI as the case insensitive flag `i` doesn't seem to work with OpenAPI generator. As a workaround, I enumerated the different letter cases (i.e. added `A-Z`) in each of the expressions where the case insensitive flag was used.